exonum-java-binding/fakes/src/main/java/com/exonum/binding/fakes/services/service/TestService.java

```diff
@@ -62,12 +70,33 @@ protected TestSchema createDataSchema(View view) {
   @Override
   public void initialize(Fork fork, Configuration configuration) {
     TestSchema schema = createDataSchema(fork);
-    ProofMapIndexProxy<HashCode, String> testMap = schema.testMap();
+    ProofMapIndexProxy<HashCode, String> testMap = schema.initializeServiceMap();
     testMap.put(INITIAL_ENTRY_KEY, INITIAL_ENTRY_VALUE);
   }
 
   @Override
   public void createPublicApiHandlers(Node node, Router router) {
     // No-op: no handlers.
   }
+
+  @Override
+  public void beforeCommit(Fork fork) {
+    try {
+      TestSchema schema = createDataSchema(fork);
+      ProofMapIndexProxy<HashCode, String> map = schema.beforeCommitMap();
```

Comment:
Why does it use a _map_ 'beforeCommit' to store a single value under
      'before commit' key if EntryIndex will do (with a more appropriate name like
      'beforeCommitCounter')?